Documentation/building/unix-instructions.md

```diff
@@ -59,3 +59,19 @@ If you see errors along the lines of `SendFailure (Error writing headers)` you m
 ```sh
 mozroots --import --sync
 ```
+
+## Steps to Build on Ubuntu
+
+*Note: verified on Ubuntu 14.04 LTS*
+
+1. Install git `sudo apt-get install git`
+2. Clone the corefx repo `git clone https://github.com/dotnet/corefx.git`
+3. Install curl-dev `sudo apt-get install libcurl4-openssl-dev`
+4. Install icu `sudo apt-get install libicu52`
+5. Install llvm `sudo apt-get install llvm`
+6. Install lldb `sudo apt-get install lldb-3.5`
+7. Install cmake `sudo apt-get install cmake`
+8. Install clang `sudo apt-get install clang-3.6`
+9. Install libunwind `sudo apt-get install libunwind8-dev`
+10. Create a symlink for clang bins `sudo ln -s /usr/bin/clang-3.6 /usr/bin/clang && sudo ln -s /usr/bin/clang++-3.6 /usr/bin/clang++`
```

Comment:
@ellismg yes it appears as though this is needed.  Take a look at a repro on a new Ubuntu install I just performed and see verification for this requirement:

```bash
tstringer@tahr3:~/dev/corefx$ ls /usr/bin/*clang*
/usr/bin/clang-3.6    /usr/bin/clang-apply-replacements-3.6  /usr/bin/clang-query-3.6   /usr/bin/clang-tblgen-3.6
/usr/bin/clang++-3.6  /usr/bin/clang-check-3.6               /usr/bin/clang-rename-3.6  /usr/bin/clang-tidy-3.6
tstringer@tahr3:~/dev/corefx$ ./build.sh
Setting up directories for build
Checking pre-requisites...
Please install clang before running this script
tstringer@tahr3:~/dev/corefx$ sudo ln -s /usr/bin/clang-3.6 /usr/bin/clang && sudo ln -s /usr/bin/clang++-3.6 /usr/bin/clang++
tstringer@tahr3:~/dev/corefx$ ./build.sh
Setting up directories for build
Checking pre-requisites...
Installing dotnet cli...
```